src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -567,7 +567,7 @@ public static LbfgsLogisticRegressionBinaryTrainer LbfgsLogisticRegression(this
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/PoissonRegression.cs)]
+        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LbfgsPoissonRegression.cs)]
         /// ]]></format>
         /// </example>
         public static LbfgsPoissonRegressionTrainer LbfgsPoissonRegression(this RegressionCatalog.RegressionTrainers catalog,
```

Comment:
>LbfgsPoissonRegressionTrainer [](start = 22, length = 29)

there were other renamings which accompanied this .. should we fix those too in this PR ?

[https://github.com/dotnet/machinelearning/issues/3016#issuecomment-476369026](https://github.com/dotnet/machinelearning/issues/3016#issuecomment-476369026)

Reply:
Rest of the examples are not using .tt or they have been converted to .tt after the above change. So, those are fine right now.

---
In reply to: [270582198](https://github.com/dotnet/machinelearning/pull/3140#discussion_r270582198) [](ancestors = 270582198)